.github/workflows/tree-sitter-extractor-test.yml

```diff
@@ -43,4 +43,4 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Run clippy
-        run: cargo clippy -- --no-deps # -D warnings
+        run: cargo clippy -- --no-deps -D warnings -A clippy::new_without_default -A clippy::too_many_arguments
```

Comment:
We may want to address the "too many arguments" warning at some point. We could use a sort of builder pattern to configure an extractor instead of passing a lot of arguments to the "extract" method.